### PR TITLE
arjun's UC creativity essay

### PR DESCRIPTION
**Pull request by JoshiRiddhi**
the entire reason that brown zelda came into being.

### CODE REVIEW

File: Riddhi-feature [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1
+q
```
Comment (siddpuri) [inline Riddhi-feature:2]: Wait a minute, there's not supposed to be a "q".

Comment (SuenAidan) [inline Riddhi-feature:2]: Only supposed to be one line with 1

**Review by siddpuri — CHANGES_REQUESTED** [review]
You added an extra line!

**Review by SuenAidan — CHANGES_REQUESTED** [review]
You have an extra line with a q it should only be one line with 1

**Review by siddpuri — APPROVED** [review]
Yes, much better.

**Review by SuenAidan — APPROVED** [review]
good change

**Pull request merged**